tests/tests_unit/test_utils/test_time.py

```diff
@@ -35,6 +37,30 @@
     import pandas
 
 
+class TestDatetimeToMsIsoTimestamp:
+    @pytest.mark.skipif(platform.system() == "Windows", reason="Overriding timezone is too much hassle on Windows")
+    def test_timezone_unaware(self):
+        input_datetime = datetime(2021, 1, 1, 0, 0, 0, 0)
+        with tmp_set_envvar("TZ", "CET"):
+            time.tzset()
+            assert datetime_to_ms_iso_timestamp(input_datetime) == "2021-01-01T00:00:00.000+01:00"
+
+    def test_timezone_cet(self):
+        input_datetime = datetime(2021, 1, 1, 0, 0, 0, 0, tzinfo=pytz.timezone("CET"))
```

Comment:
This example - and if the pytz example has been cloned from somewhere else - should be changed, reason being:

> The main pitfall of using the `pytz` package is that it requires you to use the `localize` method to correctly attach timezone information to naive `datetime` objects. Simply setting the `tzinfo` attribute directly often leads to incorrect timezone behavior, such as wrong offset calculations due to daylight saving time transitions.

Example:
```
import datetime
import pytz

eastern = pytz.timezone("US/Eastern")

user_date = eastern.localize(datetime.datetime(2020, 4, 15, 6, 0, 0))
utc_date = user_date.astimezone(pytz.utc)
print(utc_date)  # 2020-04-15 10:00:00+00:00
```

Reply:
Something like this?